tests/test_synology_dsm.py

```diff
@@ -716,3 +718,21 @@ def test_surveillance_camera(self):
         assert self.api.surveillance_station.get_home_mode_status()
         assert self.api.surveillance_station.set_home_mode(False)
         assert self.api.surveillance_station.set_home_mode(True)
+
+    def test_shares(self):
+        """Test shares."""
+        assert self.api.share
+        self.api.share.update()
+        assert self.api.share.shares
+        for share_name in self.api.share.shares_names:
```

Comment:
You should update this to the latest changes

Reply:
Done